ibis/formats/__init__.py

```diff
@@ -45,6 +45,11 @@ def to_ibis(cls, typ: T, nullable: bool = True) -> DataType:
         Ibis DataType.
         """
 
+    @classmethod
+    def to_string(cls, dtype: DataType) -> str:
```

Comment:
We could have a from_string as well which is basically "parse the backend specific string", we have multiple "parse" implementations scatterend in the codebase actually.